go-controller/pkg/ovn/ipallocator/bitmap.go

```diff
@@ -0,0 +1,197 @@
+package ipallocator
```

Comment:
copied from https://github.com/kubernetes/kubernetes/blob/master/pkg/registry/core/service/allocator/bitmap.go ? If so maybe add a comment stating that and that we dont need to unit test it? Or should you also copy in their unit test so our coverage doesn't fall?

Reply:
In the new PR, I have included the unit tests as well. Please see PR #1374